.ci/static-checks.sh

```diff
@@ -385,7 +385,7 @@ check_docs()
 	if [ ! "$(command -v $cmd)" ]
 	then
 		info "Installing $cmd utility"
-		go get -u "mvdan.cc/xurls/cmd/$cmd"
+		go get -u "github.com/mvdan/${cmd}"
```

Comment:
Ah - actually, this needs to be:

```suggestion
		go get -u "github.com/mvdan/xurls/cmd/${cmd}"
```

Reply:
When I saw this last week, I could not make the github URL work. The 'mvdan.cc' site seemed to be temporarily down, and then came back up, and everything started working for me again, hence I didn't file a fix or dig deeper on how to make the github URLs work.